docs/user/guides/subprojects.rst

```diff
@@ -0,0 +1,32 @@
+How To Add a Subproject
```

Comment:
Seems this should be `Manage` a project, with a section on adding, removing, and aliases?